### ISSUE TITLE
Compatibility with dragscroll plugin

### ISSUE THREAD

**Issue by kpagcha** [teijo/jquery-bracket#108]
I am loading a bracket in a div that takes the left half of the main container. To explore the bracket, I am using the [dragscroll ](https://github.com/asvd/dragscroll) plugin. Dragging horizontally moves the bracket as expected.

Here is a working [example](https://plnkr.co/edit/ManaRsLCXhslEK5cz9tt).

The issue comes when trying this on mobile. Dragging the brackets just won't work, I cannot figure out why so I am here to hopefully get some help. I know this is not a issue directly related to the bracket plugin, but I have a suspicion that that dragscroll does not work on mobile because of the DOM generated by the plugin, so that makes it related to it.

**Comment by teijo** [comment]
Does it make any difference if you wrap the bracket container instead of using the dragscroll onto the container itself? 

Now:
```html
<div id="bracket" class="col-md-6 dragscroll" style="overflow: hidden;"></div>
```

Test:
```html
<div class="col-md-6 dragscroll" style="overflow: hidden;">
  <div id="bracket"></div>
</div>
```

Or try applying dragscroll onto the container after the bracket is rendered once?

**Comment by kpagcha** [comment]
Yeah I've tried that but it really didn't work. [Check it out](https://plnkr.co/edit/i4W2bsxJyI3dDOzWHrJa?p=preview).

As of applying the dragscroll after the bracket is rendered, yeah, I thought of that; you can reapply the dragscroll pluggin by calling `dragscroll.reset()` but it did not help either. And it doesn't make sense either, because the dragscroll is actually working (so the functionality is being applied to the rendered elements: the bracket), it just does not work on mobile
